src/github.com/matrix-org/dendrite/syncapi/sync/requestpool.go

```diff
@@ -93,7 +93,7 @@ func (rp *RequestPool) OnIncomingSyncRequest(req *http.Request, device *authtype
 		case <-timer.C:
 			return util.JSONResponse{
 				Code: 200,
-				JSON: types.NewResponse(syncReq.since),
+				JSON: types.NewResponse(currPos),
```

Comment:
is this an unrelated fix? looks ok, just wondering.

Reply:
I otherwise would need to dereference `syncReq.since`, though in practice it probably doesn't really matter which one we chose─other than using `currPos` might be a bit faster.